keyboard_handler/test/fake_player.hpp

```diff
@@ -27,10 +27,10 @@ class FakePlayer : public std::enable_shared_from_this<FakePlayer>
 
   void register_callbacks(KeyboardHandler & keyboard_handler)
   {
-    std::weak_ptr<FakePlayer> player_weak_ptr(shared_from_this());
-    auto callback = [player_weak_ptr](KeyboardHandler::KeyCode key_code,
-        KeyboardHandler::KeyModifiers key_modifiers) {
-        auto player_shared_ptr = player_weak_ptr.lock();
+    std::weak_ptr<FakePlayer> weak_ptr(shared_from_this());
```

Comment:
@clalancette I am curious about what was an original error message from the new version of the uncrustify?
It seems renames is an extra step here. 
It seems moving first line of lambda on the new line should be enough.
e.g.
```
auto callback = [player_weak_ptr](KeyboardHandler::KeyCode key_code,
```
```
auto callback = 
  [player_weak_ptr](KeyboardHandler::KeyCode key_code,
```
I would like to keep names if possible.

Reply:
Unfortunately, it is hard to make that compatible with both old and new uncrustify.

As far as I can tell, uncrustify 0.78 (in Ubuntu 24.04) just indents lambdas differently from 0.72 (in Ubuntu 22.04).  And despite some deep debugging in uncrustify itself, I have not found a way to make them the same.  So the best thing I've been able to find to satisfy both of them is to shorten things up so that the lambda signatures are on the same line.